kubejs/server_scripts/mods/modern_industrialization.js

```diff
@@ -46,6 +47,21 @@ ServerEvents.recipes(e => {
         mi('electric_age/component/craft/op_amp_asbl')
     ];
     MI_DELETED_ITEMS.forEach(id => e.remove( {id: id} ));
+
+    // -- BOSS CRUSHER -- // 
+    e.shaped(mi('boss_crusher'), [
+        'RCR',
+        'PHP',
+        'MCM'
+    ],
+    {
+        H: mi('chemically_inert_ptfe_casing'),
```

Comment:
If you want it in HV, let's solidify it by changing this PTFE casing to a Turbo Machine Hull